README.md

```diff
@@ -40,9 +39,11 @@ module "postgresql_rds" {
 
 ## Variables
 
+- `vpc_id` - ID of VPC meant to house database
+
```

Comment:
Extra space here makes the rendered output look a little strange:

<img width="701" alt="terraform-aws-postgresql-rds_readme_md_at_ded6818337164b1377dc3820505b6a6e135b2401_ _azavea_terraform-aws-postgresql-rds" src="https://cloud.githubusercontent.com/assets/43639/17519696/04164062-5e1b-11e6-93e8-6d253ce9f325.png">